vector/src/main/java/im/vector/app/core/platform/VectorBaseActivity.kt

```diff
@@ -500,7 +501,7 @@ abstract class VectorBaseActivity<VB : ViewBinding> : AppCompatActivity(), Maver
     /**
      * Configure the Toolbar, with default back button.
      */
-    protected fun configureToolbar(toolbar: MaterialToolbar, displayBack: Boolean = true) {
```

Comment:
currently just removed default value to not cause compile error. Later should be replaced completely by builder method